R/Switzerland.R

```diff
@@ -94,35 +69,79 @@ Switzerland <- R6::R6Class("Switzerland",
       )
     },
 
+    #' @description Download function to get raw data. Downloads
+    #' the updated list of CSV files using `download_JSON`, filters
+    #' that to identify the required CSV files, then uses the parent
+    #' method `download` to download the CSV files.
+    #' @importFrom purrr keep
+    download = function() {
+      message_verbose(
+        self$verbose,
+        paste0("Downloading updated URLs from ", self$common_data_urls$main))
+
+      super$download_JSON()
+
+      self$data_urls <-
+        self$data$raw$main$data$sources$individual$csv$daily %>%
+        keep(names(.) %in% c("cases", "test", "death", "hosp"))
+
+      super$download()
+    },
+
     #' @description Switzerland specific state level data cleaning
-    #' @importFrom dplyr select filter mutate left_join rename
+    #' @importFrom dplyr select filter mutate left_join rename full_join
     #' @importFrom lubridate as_date ymd
     #' @importFrom rlang .data
```

Comment:
I think based on #437 we may not need this (for mysterious reasons).

Reply:
No, you do need it here :slightly_smiling_face:. The lines are removed in d3c150b155c1f56689f7c380c9c952db7941e3c3 are different and were interpreted as a re-import by roxygen2.

Reply:
(Well, maybe not here specifically since it doesn't look like your using `.data` in this function but it needs to be somewhere.)

Reply:
Ha okay my bad. Thanks Hugo

Reply:
We do use it at lines 134-138 - though I'm not sure if it's necessary.